src/audio/pipewire.c

```diff
@@ -128,8 +128,8 @@ static void on_process(void *userdata)
     buf->datas[0].chunk->size = number_of_bytes;
     buf->datas[0].chunk->stride = state->stride;
     pw_stream_queue_buffer(state->stream, b);
-    // signal to the main thread that data can be written again
-    spa_system_eventfd_write(pw_thread_loop_get_loop(state->loop)->system, state->eventfd_number, 42);
+    // signal to the main thread that data can be written again, make sure it's woken only once
+    spa_system_eventfd_write(pw_thread_loop_get_loop(state->loop)->system, state->eventfd_number, 1);
```

Comment:
? This is not a number of wakeups, since you have not passed the SPA_FD_EVENT_SEMAPHORE flag on event creation

Reply:
then, does the value matter? I put 1 in there now, so I don't get as many client missed 1 wakeups warnings, though I still do from time to time. As far as I remember, I deleted the 42 part of the comment

Reply:
The value does not matter since you don't use it when waiting for the event.

> I put 1 in there now, so I don't get as many client missed 1 wakeups warnings

Did you really measure a difference, or is that just an impression?